canonical/jaas-dashboard
# WD-3028 - Update click handlers to use links

## Done

- Update navigation events to be accessible links.
- Update panel open/close events to not be included in the history .

## QA

- Pull code
- Run `dotrun clean && dotrun serve`
- Open http://0.0.0.0:8036/
- Navigate through the app and check that links/buttons that cause navigation/url changes are links (you should be able to right click and open in a new tab).

## Details

https://warthogs.atlassian.net/browse/WD-3028
Fixes: #1243.
